.github/ISSUE_TEMPLATE/Bug_report.md

```diff
@@ -0,0 +1,29 @@
+---
+name: Bug report 🐛
+about: You're having technical issues
+labels: 'bug'
+---
+
+<!--- Please fill out the template to the best of your ability -->
+
+## Expected Behavior
+<!--- What should have happened? -->
+
+## Current Behavior
+<!--- What went wrong? -->
+
+## Possible Solution
+<!--- (Not obligatory) Suggest a fix/reason -->
+
+## Steps to Reproduce
+<!--- Please provide a clear sequence of steps to reproduce this bug -->
+<!--- Include code and images, if relevant -->
+1.
+2.
+3.
+4.
+
+## Environment
+- JS SDK Version: <!--- E.g. 7.1.0 -->
+- Installation Method: <!-- I.e. NPM/Yarn or <script> import -->
+- Browser and Version: <!-- E.g. Chrome 84-->
```

Comment:
This repo is shared for both of browser and node should this template cover both of those two?  Or might be we can improve this later when we have node.

Reply:
Yeah we can have another set for node, or just add another section to ask environment details.